### DISCUSSION TITLE
Silencing Request Responses

### DISCUSSION THREAD

**Discussion by tahir-ijaz** [ucfopen/canvasapi#516] [Q&A]
First of all, thank you so much for _**canvasapi**_, its really awesome!

Perhaps I am doing something wrong. I was wondering if there is a way I could silence request responses being logged to stdout. 

For instance, when I try to traverse through enrollments:

```python
for course in account.get_courses(enrollment_term_id=4, per_page=100):
        for enrollment in course.get_enrollments(type="StudentEnrollment", per_page=100):
```

stdout gets flooded with following:

```python
...
Request: GET https://xxxx.instructure.com/api/v1/courses/448/enrollments
Response: GET https://xxxx.instructure.com/api/v1/courses/448/enrollments 200
Request: GET https://xxxx.instructure.com/api/v1/courses/449/enrollments
Response: GET https://xxxx.instructure.com/api/v1/courses/449/enrollments 200
Request: GET https://xxxx.instructure.com/api/v1/courses/450/enrollments
Response: GET https://xxxx.instructure.com/api/v1/courses/450/enrollments 200
Request: GET https://xxxx.instructure.com/api/v1/courses/451/enrollments
Response: GET https://xxxx.instructure.com/api/v1/courses/451/enrollments 200
...
```

I tried setting the logging level to _INFO -> WARNING -> ERROR_ as well as _disabling_ it all together without any success.

Would really appreciate any ideas or suggestions!

**Comment by Thetwam** [comment]
Are you using the Python logger for anything else? I suspect something about your setup is triggering CanvasAPI.

Here's one way of turning logs on, it might provide some insight as to what you've enabled:

https://canvasapi.readthedocs.io/en/stable/debugging.html
